Plutarch/Convert.hs

```diff
@@ -15,21 +15,7 @@ module Plutarch.Convert (
 ) where
 
 import GHC.Generics (Generic)
-import Plutarch.Builtin.Bool (PBool (PFalse, PTrue))
-import Plutarch.Builtin.Integer (PInteger)
-import Plutarch.ByteString (PByteString)
-import Plutarch.Internal.Eq (PEq)
-import Plutarch.Internal.Newtype (PlutusTypeNewtype)
-import Plutarch.Internal.Ord (POrd)
-import Plutarch.Internal.Other (pto)
-import Plutarch.Internal.PLam (plam)
-import Plutarch.Internal.PlutusType (
-  DPTStrat,
-  DerivePlutusType,
-  PlutusType,
-  pcon,
- )
-import Plutarch.Internal.Term (S, Term, (#), (:-->))
+import Plutarch.Internal (DPTStrat, DerivePlutusType, PBool (PFalse, PTrue), PByteString, PEq, PInteger, POrd, PlutusType, PlutusTypeNewtype, S, Term, pcon, plam, pto, (#), (:-->))
```

Comment:
Could we roll this entire module in with `PByteString`? I think the deps should be breakable.

Reply:
yes possibly. this PR's gone extremely big and I was very fed up near end. More can be done definitely. This is good enough for #767

Reply:
I'll break it into another ticket.